metaflow/plugins/argo/argo_workflows.py

```diff
@@ -513,6 +512,12 @@ def _compile_workflow_template(self):
                 WorkflowSpec()
                 # Set overall workflow timeout.
                 .active_deadline_seconds(self.workflow_timeout)
+                # Remove workflow after a week.
+                .ttl_strategy(
+                    {
```

Comment:
This would only remove the workflow after a week, but leave the pod object untouched. I think we should leave the ttl_strategy for workflows as is but introduce ttl strategy within the container template and the sensor object.

Reply:
I think if you delete the workflow the corresponding resources will be deleted too. 

Don't have a reference at hand, but pretty confident this is the case.